monkey/monkey_island/cc/ui/src/components/report-components/security/PostBreach.js

```diff
@@ -62,19 +63,26 @@ class PostBreachComponent extends React.Component {
     });
     let defaultPageSize = pbaMachines.length > pageSize ? pageSize : pbaMachines.length;
     let showPagination = pbaMachines > pageSize;
+    const howManyPBAs = pbaMachines.reduce((accumulated, pbaMachine) => accumulated+pbaMachine["pba_results"].length,0)
     return (
-      <div className="data-table-container">
-        <ReactTable
-          columns={columns}
-          data={pbaMachines}
-          showPagination={showPagination}
-          defaultPageSize={defaultPageSize}
-          SubComponent={row => {
-            return renderDetails(row.original.pba_results);
-          }}
-        />
-      </div>
-
+      <>
```

Comment:
what is this?

Reply:
https://reactjs.org/docs/fragments.html#short-syntax

Reply:
Nice, didn't know this.